src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,44 @@
+.stopwatch {
+  position: relative;
+  width: $stopwatchSize;
+  height: $stopwatchSize;
+  background-color: rgb(235, 245, 145);
+  border: 2px solid yellow;
+  border-radius: 50%;
+
+
+  &__dot {
+    position: absolute;
+    top: 50%;
```

Comment:
also to center the element you need to use transform: translate(-50%, -50%). Because absolute left: 50% places the element exactly in the middle of the parent element, but children ( dot in your case) has its own width, in this case you need to move dot by half the width. The same for height